src/constant.js

```diff
@@ -6,4 +6,5 @@ export const PATHS = {
   CLASS: "/class",
   COURSE: "/course",
   COURSE_CONTENT: "/course/:courseId",
+  CLASSLIST: "/classList",
```

Comment:
I think we can remove the `CLASSLIST`. We might not need it.

Reply:
okay